assets/swagger.json

```diff
@@ -4899,6 +4899,10 @@
             "$ref": "#/definitions/v1alpha1HelmParameter"
           }
         },
+        "passCredentials": {
+          "type": "boolean",
+          "title": "PassCredentials pass credentials to all domains (Helm's --pass-credentials)"
+        },
```

Comment:
Where does this come from?

Reply:
Thanks for catching this. Not sure how this got added. I have removed this part.